src/submission-report/submission-report.service.ts

```diff
@@ -1,5 +1,7 @@
 import { Injectable, HttpStatus } from '@nestjs/common';
 import { EaseyException } from '@us-epa-camd/easey-common/exceptions';
+import { withSlaveConnection } from '@us-epa-camd/easey-common/connection';
```

Comment:
This endpoint may actually be fine, but I'd vote for making this use the primary connection: there's a chance an admin could query submissions immediately after a submission was made, and it could be that the lag causes it to not be reflected in the report. As an "admin" module, I think this should access the most recent data.